lambdas/instant-sync-events/src/clients.ts

```diff
@@ -0,0 +1,78 @@
+import knex, { Knex } from 'knex';
+import { fetchSecret } from '@pocket-tools/lambda-secrets';
+import { config } from './config';
+
+let readDb: Knex;
+let writeDb: Knex;
+
+/**
+ * Create a db client for reads from readitla_ril-* databases
+ */
+export async function readClient(): Promise<knex.Knex<any, any[]>> {
+  if (readDb) return readDb;
+
+  const secret = await fetchSecret(config.dbSecretName);
+
+  readDb = createConnection({
+    host: secret.read_host,
+    port: 3306,
```

Comment:
is this not in the secret?

Reply:
no lol idk i feel a db port is alweays the same